src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../../global.css';
+import { FcManager } from 'react-icons/fc';
+
+function NavBar() {
+  return (
+    <div className="Navbar">
+      <div className="leftSide">
+        <input type="text" placeholder="Search" />
+      </div>
+      <div className="rightSide">
+        <FcManager size="60" />
+      </div>
+    </div>
```

Comment:
I recommend using styled-components instead of adding the classes to the component itself and, remove the CSS from the global one.
I.e.:
NavBar.styles.js
```js
const StyledNavBar = styled.div`
  background-color: rgb(13, 13, 71);
  width: 100%;
  height: 80px;
  display: flex;
`;

export { StyledNavBar };
```

NavBar.component.jsx
```jsx
function NavBar () {
  return (
    <StyledNavBar>
    // Other components
    </StyledNavBar>
  )
}
```

Using styled-components is one solution for keeping the control of which CSS styles which component.